lib/models/fsm-transition.js

```diff
@@ -578,11 +578,15 @@ function transition(opts, cb) {
 
                 opts.moray.batch(requests,
                     function batchCb(batchErr, batchMeta) {
+                    opts.log.debug({
+                        requests: requests,
+                        batchErr: batchErr,
+                        batchMeta: batchMeta
+                    }, 'transition: removeTransitions batch');
```

Comment:
Intentionally added some more logging every time we ran moray batches to help debugging little bit faster.